scripts/constants.sh

```diff
@@ -6,16 +6,25 @@ GOPATH="$(go env GOPATH)"
 # Avalabs docker hub
 dockerhub_repo="avaplatform/avalanchego"
 
-# Current branch
-current_branch=${CURRENT_BRANCH:-$(git describe --tags --exact-match 2> /dev/null || git symbolic-ref -q --short HEAD || git rev-parse --short HEAD)}
-echo "Using branch: ${current_branch}"
+# If we don't have git installed locally or this isn't a git repository (say building from a release), don't set our git constants.
+if [ ! git --version &> /dev/null ] || [ ! -d .git ]
```

Comment:
Can we remove this check in favor of assuming git is installed?

Reply:
Discussed offline. Seems like a reasonable requirement.